django/db/models/functions/text.py

```diff
@@ -155,6 +156,8 @@ class MD5(Transform):
     lookup_name = 'md5'
 
     def as_oracle(self, compiler, connection, **extra_context):
+        if self.function == 'SHA224':
+            raise NotSupportedError('SHA224() is not supported on Oracle.')
```

Comment:
This would require duplicating things for `SHA256`, `SHA384` and `SHA512` because they inherit from `SHA224` to avoid duplicating things for the MySQL case... So I've gone with the following:
```python
    def as_oracle(self, compiler, connection, **extra_context):
        if self.function != 'SHA224':
            return super().as_oracle(compiler, connection, **extra_content)
        raise NotSupportedError('SHA224 is not supported on Oracle.')
```
Another possibility is to add the following line to those three classes:
```python
    as_oracle = SHA1.as_oracle
```
If you'd prefer that (with a comment) then I can change...

Reply:
Ah, ok I missed that. Looks good now :+1: